File: app/src/main/java/com/rtukpe/fixtures/data/model/Player.java
package com.rtukpe.fixtures.data.model;

import com.google.gson.Gson;
import com.google.gson.annotations.SerializedName;

public class Player {

    public int id;
    public String name;
    public String position;
    @SerializedName("jerseyNumber")
    public int jerseyNumber;
    @SerializedName("dateOfBirth")
    public String dateOfBirth;
    public String nationality;

    @Override
    public String toString() {
        return new Gson().toJson(this);
    }
}

File: app/src/main/java/com/rtukpe/fixtures/data/model/Player.kt
package com.rtukpe.fixtures.data.model

import com.google.gson.Gson
import com.google.gson.annotations.SerializedName

class Player {
    @JvmField
    var id: Int = 0
    lateinit var nationality: String
    @SerializedName("dateOfBirth")
    lateinit var dateOfBirth: String
    @SerializedName("jerseyNumber")
    @JvmField
    var jerseyNumber: Int = 0
    lateinit var position: String
    lateinit var name: String

    override fun toString(): String {
        return Gson().toJson(this)
    }
}
